### DISCUSSION TITLE
issue with climate.set_temperature

### DISCUSSION THREAD

**Discussion by phsdv** [custom-components/pyscript#523] [Q&A] [answered]
In the thermostat example here: [(https://github.com/custom-components/pyscript/wiki/Thermostat-adjustment-using-Occupancy)] it says:
`climate.set_temperature(entity_id="climate.radio_thermostat_company_of_america_ct101_thermostat_iris_mode",target_temp_low=lowtemp,target_temp_high=hightemp,hvac_mode="heat_cool")`

However when I do something similar like this:
`climate.set_temperature(entity_id="climate.smart_kantoor_vloerverwarming", target_temp_high=SetHigh, hvac_mode="heat")`

I get the following error:
`Exception in <file.office_heating.thermostat_office_sethigh> line 72: climate.set_temperature(entity_id="climate.smart_kantoor_vloerverwarming", target_temp_high=SetHigh, hvac_mode="heat") ^ AttributeError: 'bool' object has no attribute 'set_temperature'`

Am I doing something wrong and/or is there an error in the example on the wiki page?
Thanks
Paul

**Comment by phsdv** [comment, marked as answer]
Found it my self. I used a boolean called climate that signals if it should switch on or off the heater. 
```
    if hour >= 8 and hour <= 16:       # working hours
        climate = True
    if climate:
        log.warning(f"Thermostat set to SetHigh at {trigtime}, temp_inside={temperatureInside}, temp_outside={temperatureOutside}")
        #climate.set_temperature(entity_id="climate.smart_kantoor_vloerverwarming", temperature=SetHigh)
        climate.set_temperature(entity_id="climate.smart_kantoor_vloerverwarming", temperature=SetHigh)
```
stupid mistake
[edited block for better visibility]

**Comment by ALERTua** [comment]
You can mark code blocks using three apostrophes at the start of a block and three apostrophes after the block.
```
my code block
    with spaces
```
![image](https://github.com/custom-components/pyscript/assets/8375129/a98510fa-e169-4d18-a680-70f4319fdbc3)
